### PR TITLE
Find and replace unicode arrows

### PR DESCRIPTION
**Pull request by duanebester**

### CODE REVIEW

**Comment by duanebester** [comment]
Seeing the following in the CI logs if anyone can advise? -
```
The command "if [ "$SCALAENV" = "jvm" ]; then sbt ++$TRAVIS_SCALA_VERSION $SBT_JAVA_OPT validateJVM; elif [ "$SCALAENV" = "js" ]; then sbt ++$TRAVIS_SCALA_VERSION $SBT_JAVA_OPT compile; sbt ++$TRAVIS_SCALA_VERSION $SBT_JAVA_OPT validateJS; elif [ "$G4SBUILD" = "docs" ]; then sbt ++$TRAVIS_SCALA_VERSION $SBT_JAVA_OPT docs/tut; else echo "You might not be invited to the party"; fi" exited with 1.
```

File: github4s/shared/src/main/scala/github4s/HttpClient.scala [hunk 1]
```diff
@@ -80,8 +80,8 @@ class HttpRequestBuilder[C, M[_]](
 
   def withAuth(accessToken: Option[String] = None) = {
     val authHeader = accessToken match {
-      case Some(token) ⇒ Map("Authorization" → s"token $token")
-      case _           ⇒ Map.empty[String, String]
+      case Some(token) => Map("Authorization" → s"token $token")
```
Comment (BenFradet) [inline github4s/shared/src/main/scala/github4s/HttpClient.scala]: should we to take care of `→` as well? maybe in a separate PR

Comment (duanebester) [reply github4s/shared/src/main/scala/github4s/HttpClient.scala]: Was planning on a separate PR, i'll get a ticket started! Thanks

File: github4s/jvm/src/test/scala/github4s/unit/ApiSpec.scala [hunk 2]
```diff
@@ -400,15 +400,10 @@ class ApiSpec
...
-  it should "return an empty list when an invalid ref is passed" in {
+  it should "return an error when an invalid ref is passed" in {
     val response =
       repos.listStatuses(accessToken, headerUserAgent, validRepoOwner, validRepoName, invalidRef)
-    response should be('right)
-
-    response.toOption map { r ⇒
-      r.result.isEmpty shouldBe true
-      r.statusCode shouldBe okStatusCode
-    }
```
[the remark this answers is not in the document]
Comment (duanebester) [reply github4s/jvm/src/test/scala/github4s/unit/ApiSpec.scala]: Yes, sure thing

[the remark this answers is not in the document]
Comment (duanebester) [reply github4s/jvm/src/test/scala/github4s/unit/ApiSpec.scala]: @BenFradet 
When I checkout this main repo and run sbt validateJVM it fails

```
$ git clone git@github.com:47deg/github4s.git
$ cd github4s/
$ sbt validateJVM
# ...
[info] - should return an empty list when an invalid ref is passed *** FAILED ***
[info]   Left(github4s.GithubResponses$UnsuccessfulHttpRequest: Failed invoking with status : 404 body : 
[info]    ) was not right (ApiSpec.scala:406)
# ...
[info] Tests: succeeded 229, failed 1, canceled 0, ignored 101, pending 0
[info] *** 1 TEST FAILED ***
[error] Failed tests:
[error] 	github4s.unit.ApiSpec
[error] (Test / test) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 60 s, completed Nov 28, 2019, 1:06:14 PM
```

[the remark this answers is not in the document]
Comment (BenFradet) [reply github4s/jvm/src/test/scala/github4s/unit/ApiSpec.scala]: this test is very flaky, it works for me locally and on travis in master

[the remark this answers is not in the document]
Comment (BenFradet) [reply github4s/jvm/src/test/scala/github4s/unit/ApiSpec.scala]: logged #300 in this regard

**Pull request closed without merging**